### ISSUE TITLE
SteamFriends.ActivateGameOverlay("achievements") points to a broken link

### ISSUE THREAD

**Issue by Tanek81** [rlabrecque/Steamworks.NET#565]
As the title suggests, tested with the latest plugin version (20.2.0), the call points the Steam client to a wrong URL:

- Link provided by the plugin: https://steamcommunity.com/profiles/0/stats/appid/<<APP_ID>>/achievements/
- Actual link from the Steam UI: https://steamcommunity.com/id/<<USER_ID>>/stats/<<APP_ID>>/achievements/

I suspect this URL changed during last month Steam UI update roll out, the fact that no one at Valve thought that the old URL should have a redirect baffles me, but in the mean time all calls in production are returning a broken link.

**Comment by Tanek81** [comment]
I wanted to fix it myself and do a pull request but I couldn't find the piece of code where that link is generated in the native code, the native method is "NativeMethods.ISteamFriends_ActivateGameOverlay".
Can anyone help so that we can fix this in prod?🤚

**Comment by rlabrecque** [comment]
Sorry for not getting to respond to this; but this sounds like a bug in the steam client itself unfortunately. All Steamworks.NET does is calls the `ISteamFriends_ActivateGameOverlay` function in steam_api.dll, while ensuring that the arguments we pass are valid(ish), then steam_api.dll largely just is finding the active Steam instance and calling into it.

I wonder if this is still broken? :/

**Comment by Tanek81** [comment]
Hey Riley, thank you for your reply!
You are right, I just checked our game in production and the link for the achievements now works and opens the new UI page in the Steam overlay. We haven't made any update/change, so probably as you said this was fixed in a later Steam client update.

On a different note, unfortunately the Leaderboards link is broken, I can't remember if it was already broken when I did these checks or if it got broken later.
